includes/core/classes/class-assets.php

```diff
@@ -310,12 +310,14 @@ protected function localize( int $post_id ): array {
 				'dateFormat'           => $settings->get_value( 'general', 'formatting', 'date_format' ),
 				'enableAnonymousRsvp'  => ( 1 === (int) $settings->get_value( 'general', 'general', 'enable_anonymous_rsvp' ) ),
 				'enableInitialDecline' => ( 1 === (int) $settings->get_value( 'general', 'general', 'enable_initial_decline' ) ),
+				'mapPlatform'          => $settings->get_value( 'general', 'general', 'map_platform' ),
 				'maxAttendanceLimit'   => $settings->get_value( 'general', 'general', 'max_attendance_limit' ),
 				'maxGuestLimit'        => $settings->get_value( 'general', 'general', 'max_guest_limit' ),
 				'showTimezone'         => ( 1 === (int) $settings->get_value( 'general', 'formatting', 'show_timezone' ) ),
 				'timeFormat'           => $settings->get_value( 'general', 'formatting', 'time_format' ),
 			),
 			'urls'         => array(
+				'pluginUri'       => GATHERPRESS_CORE_URL,
```

Comment:
everything uses `Url` vs `Uri`. Let's change to `Url` to be consistent.

Reply:
Done!